packages/zendesk-adapter/src/filters/handle_template_expressions.ts

```diff
@@ -137,7 +141,8 @@ const seekAndMarkPotentialReferences = (formula: string): string => {
 // This function receives a formula that contains zendesk-style references and replaces
 // it with salto style templates.
 const formulaToTemplate = (formula: string,
-  instancesByType: Record<string, InstanceElement[]>): TemplateExpression | string => {
+  instancesByType: Record<string, InstanceElement[]>,
+  enableMissingReferences: boolean | undefined): TemplateExpression | string => {
```

Comment:
nit (I know some of this is existing):
```suggestion
nit:
```suggestion
const formulaToTemplate = (
  formula: string,
  instancesByType: Record<string, InstanceElement[]>,
  enableMissingReferences?: boolean,
): TemplateExpression | string => {
```
```